package.json

```diff
@@ -2,15 +2,15 @@
     "name": "stan-vscode",
     "displayName": "stan-vscode",
     "description": "Stan language support for VS Code",
-    "version": "0.2.1",
+    "version": "0.2.2",
     "publisher": "ivan-bocharov",
     "repository": "https://github.com/bocharov-ivan/stan-vscode",
     "icon": "img/stan_logo.png",
     "engines": {
         "vscode": "^1.5.0"
     },
     "categories": [
-        "Languages"
+        "Programming Languages", "Snippets"
```

Comment:
This is the new format from vscode. Instead of "Languages", we use "Programming Languages".